canonical/ubuntu.com
# Remove mkto classes

## Done
- Removed `mkto` classes including `mktoLogicalField`,`mktoCheckboxList`,`mktFormCheckbox`,`mktoHasWidth`, `mktFormReq`,`mktField`,`mktoField`,` mktoEmailField`,`mktoRequired`,` mktLblRight`,`mktoFieldDescriptor`, `marketo-Form`,`mktoLabel`,`mktoInvalid`

## QA
- Check out this feature branch
- Run the site using the command `./run serve` or `dotrun`
- View the site locally in your web browser at: http://0.0.0.0:8001/
- Run through the following [QA steps](https://canonical-web-and-design.github.io/practices/workflow/qa-steps.html)
- Please check all the forms are submitting properly

## Issue / Card

Fixes [#4535](https://github.com/canonical-web-and-design/web-squad/issues/4439)
Refer to this [epic](https://github.com/canonical-web-and-design/web-squad/issues/4436)
